### ISSUE TITLE
Missing translation in the options (german).

### ISSUE THREAD

**Issue by prsm3** [ppy/osu-stable-issues#1049]
There are 6 missing translations in the german language options!

1. "Don't change dim level during breaks"
2. "Show leaderboard during gameplay"
3. "Minimise on alt-tab during fullscreen"
4. "Render at native resolution"
5. "Audio compatibility mode"
6. "Seasonal backgrounds"

and their descriptions as well aren't translated to german.

For the 1st one I would propose:
"Hintergrund-Dimmer in den Pausen nicht ändern"
with the description:
"Macht es unmöglich den Hintergrund in Pausen zu erkennen. Nicht empfohlen."

For the 2nd one I would propose:
"Zeige Bestenliste während des Spielens"
and for the 2nd description:
"Wirkt sich auf Solo- und Multiplayer aus."

For the 3rd one I would propose:
"Minimiert das Spiel bei raustabben"

For the 4th one I would propose:
"Rendern auf native Auflösung"
For this one there is already a correct description

For the 5th one I would propose:
"Audio-Kompatibilitätsmodus"
with the description:
"Benutzt alte Audio-Engine (höhere Latenz aber erhöht Kompatibilität)"

For the 6th one I would propose:
"Saisonale Hintergründe"
with the drop-down menu entries:
- "Sometimes" -> "Manchmal"
- "Never" -> "Nie"
- "Always" -> "Immer"

**Comment by prsm3** [comment]
If there is a way I can change them myself just let me know I will gladly do that!

**Comment by Nivalyx** [comment]
the translation for osu-stable is in a "locked" state as of now unfortunately, as in no further translations could be removed, added, or modified ([due to time and technical constraints mainly](http://github.com/ppy/osu-stable-issues/issues/1032#issuecomment-1241438814)).

**Comment by Nivalyx** [comment]
yes i know there are so many things that i'm also dying to fix on my language (id) but i don't think it's possible unless this gets set to a high priority issue or anything like that 😦

**Comment by prsm3** [comment]
Okay I guess I have to wait for lazer then. Still thanks for the help!
